assets/scss/02-molecules/_arrow-nav.scss

```diff
@@ -62,20 +63,25 @@
       }
     }
   }
-  
+
   &__link {
     @media ($bp-small-max) {
       @include ma-visually-hidden;
     }
   }
 
+  &__text {
+    font-weight: 700;
+    color: #14558f;

```

Comment:
you need to include the following scss for mobile here
`@media ($bp-small-max) {
      @include ma-visually-hidden;
    }`